### PR TITLE
PR: Restore previous content of second section in STX Next blog

### PR DESCRIPTION
**Pull request by ccordoba12**
Hey @Michael-Wisniewski, sorry for changing your testimonial before consulting the changes with you first. I apologize for that.

When I was giving your blog a last read before publication, I noticed an inconsistency between the second section's title and its content. So I decided to make things match grammatically. But @juanis2112 told me you disagree with those changes.

In this pull request I restored the previous content, I changed the section's header to reflect that that's your opinions about Spyder and I'm going to leave some small suggestions to improve its content.

Thanks for your feedback and I hope things are ok now.

### CODE REVIEW

File: content/blog/STX-interview/contents.lr [hunk 1]
```diff
@@ -19,19 +19,19 @@ We are very glad to receive and share a STX Next testimonial about Spyder, along
...
+But, wouldn't it be nice to have one extra tool that could combine all the required packages into one compact working environment? Asking this question is precisely how my journey with Spyder began.
 
-In our opinion, Spyder is the go-to-solution that helps to speed up the data analysis processes. 
-For us, the biggest strength of this IDE is that it has all the benefits of Matlab/RStudio and makes them available to the Pythonista community. 
+In my opinion, Spyder is the go-to-solution that helps to speed up the data analysis processes. 
+For me, the biggest strength of this IDE is that it has all the benefits of Matlab/RStudio and makes them available to the Pythonista community. 
 It provides us with an easy-to-organize workspace that accommodates code, visualizations and shell in just one window. 
-Besides, the pane called variable explorer allows direct access to all data placed in the memory. 
+The tab, called variable explorer, allows direct access to all data placed in the memory.
```
Comment (ccordoba12) [inline content/blog/STX-interview/contents.lr]:
```suggestion
Besides, the pane called variable explorer allows direct access to all data placed in the memory. 
```

File: content/blog/STX-interview/contents.lr [hunk 2]
```diff
@@ -19,19 +19,19 @@ We are very glad to receive and share a STX Next testimonial about Spyder, along
...
-In our opinion, Spyder is the go-to-solution that helps to speed up the data analysis processes. 
-For us, the biggest strength of this IDE is that it has all the benefits of Matlab/RStudio and makes them available to the Pythonista community. 
+In my opinion, Spyder is the go-to-solution that helps to speed up the data analysis processes. 
+For me, the biggest strength of this IDE is that it has all the benefits of Matlab/RStudio and makes them available to the Pythonista community. 
 It provides us with an easy-to-organize workspace that accommodates code, visualizations and shell in just one window. 
-Besides, the pane called variable explorer allows direct access to all data placed in the memory. 
+The tab, called variable explorer, allows direct access to all data placed in the memory. 
 
-Another cool feature is the tool’s integration with Jupyter notebooks.
+Another cool feature is the tool’s integration with Jupyter.
```
Comment (ccordoba12) [inline content/blog/STX-interview/contents.lr]:
```suggestion
Another cool feature is the tool’s integration with Jupyter notebooks. 
```

File: content/blog/STX-interview/contents.lr [hunk 3]
```diff
@@ -19,19 +19,19 @@ We are very glad to receive and share a STX Next testimonial about Spyder, along
 
 ![Company Picture](STXNEXT.png)
 
-## What STX Next says about Spyder
+## What Michael says about Spyder
```
Comment (ccordoba12) [inline content/blog/STX-interview/contents.lr]: @Michael-Wisniewski, I don't know if you prefer your full name here (like in next section) or just your first name.

**Review by ccordoba12 — COMMENTED** [review]
@Michael-Wisniewski, these are my suggestions.

File: content/blog/STX-interview/contents.lr (hunk 3 above)
Comment (Michael-Wisniewski) [reply content/blog/STX-interview/contents.lr]: It is up to you. For me both Michael or Michael Wiśniewski will be fine.

**Review by Michael-Wisniewski — APPROVED** [review]
I am really happy with these changes. Thank you for update.

File: content/blog/STX-interview/contents.lr (hunk 3 above)
Comment (ccordoba12) [reply content/blog/STX-interview/contents.lr]:
```suggestion
## What Michael Wiśniewski says about Spyder
```

Ok, I'll go with your full name, for consistency with the interview section.

**Comment by ccordoba12** [comment]
These changes are now live. Thanks a lot for your patience!

**Pull request merged**